src/Microsoft.IdentityModel.Protocol.Extensions/Microsoft.IdentityModel.Protocols.xproj

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
yes the .xproj overwrites the .kproj.  This is already in the project. VS automatically adds this line every time, so I decided to check in the change so I don't get nagged all the time.